src/ReachSets/discretize.jl

```diff
@@ -1,97 +1,185 @@
 """
-    discretize(cont_sys, δ; [approx_model], [pade_expm], [lazy_expm], [lazy_sih])
+    discretize(𝑆, δ; [approx_model], [expm_method], [sih_method])
 
-Discretize a continuous system of ODEs with nondeterministic inputs.
+Apply an approximation model to `S` obtaining a discrete initial value problem.
 
-## Input
+### Input
 
-- `cont_sys`     -- continuous system
+- `𝑆`            -- initial value problem for a continuous affine ODE with
+                    nondeterministic inputs
 - `δ`            -- step size
 - `approx_model` -- the method to compute the approximation model for the
-                    discretization, among:
+                    discretization, choose among:
 
     - `forward`    -- use forward-time interpolation
     - `backward`   -- use backward-time interpolation
     - `firstorder` -- use first order approximation of the ODE
     - `nobloating` -- do not bloat the initial states
```

Comment:
These should be strings, right? I would make that explicit.

Reply:
Yes, i should use `"forward"`, etc in the docs. I doubted about using strings or symbols at some point. I guess strings is ok.